src/ui/surfaces/SiteConfiguration.tsx

```diff
@@ -98,22 +144,16 @@ function EventSettings() {
       <Form
         className="tw-pt-6 tw-max-w-170"
         defaultValues={query.data ?? {
-          collectAutomatedEvents: true,
           collectDownloads: true,
           collectEmailClicks: true,
           collectOutboundLinks: true,
-          downloadExtensions: "", // default = undefined
+          downloadExtensions: "pdf,csv,docx,xlsx,zip,doc,xls",
           useTitle: true,
           fullUrls: false,
         }}
         schema={eventSettingsSchema}
         onSubmit={mutation.mutateAsync}
       >
-        <Checkbox name="collectAutomatedEvents" 
-          label="Collect automated events"
-          helpText="It will track outbound links, email addresses clicks, and amount of downloads for common files (pdf, csv, docx, xlsx). Events will appear on your events page on simpleanalytics.com" />
-
-
         <Checkbox
```

Comment:
I think the checkboxes are not enabled by default, are they?

![telegram-cloud-photo-size-4-5850365315878340648-y](https://github.com/user-attachments/assets/56cf6819-5ff6-4632-a713-15c253c62992)

Reply:
they should be fixed by this commit https://github.com/simpleanalytics/netlify-extension/pull/10/commits/9dc0e2026a91a770d2f5a550b3eab262f0343f4f